src/main/java/net/commments/object/CommentTable.java

```diff
@@ -0,0 +1,11 @@
+package net.commments.object;
+
+public interface CommentTable {
+
+    void checkCommentInTable(int numberComment);
+
+//   String commentStatus();
```

Comment:
Please remove commented code or uncomment it.

Reply:
done